### ISSUE TITLE
ping stop method

### ISSUE THREAD

**Issue by romajstar** [silverwind/tcpie#14]
What is about to implement stop method?
I dont know how many times I will ping my server for response and I want to stop ping when it answers.
I make it this way:

```
pie.on("connect", () =>{
    pie.opts.count = 0;//to stop ping
    // great code
})
```

I think that `pie.stop()` method will be more semantic

**Comment by romajstar** [comment]
thank you for quick answer, but in readme method is called tcpie#stop, works tcpie#end

**Comment by silverwind** [comment]
Fixed in 7.0.0, it is now `.stop()`.
